### PR TITLE
apply theme update only when changed

### PR DESCRIPTION
**Pull request by namjul**
Issue: #404

### CODE REVIEW

**Comment by namjul** [comment]
@rofrischmann let me know if anything is missing.

File: packages/fela-bindings/src/ThemeProviderFactory.js [hunk 1]
```diff
@@ -20,7 +20,9 @@ export default function ThemeProviderFactory(
     }
 
     componentWillReceiveProps(nextProps: Object): void {
-      this.theme.update(nextProps.theme)
+      if (this.props.theme !== nextProps.theme) {
```
Comment (rofrischmann) [inline packages/fela-bindings/src/ThemeProviderFactory.js]: Could you still use shallowEqual here? !== checking objects might cause false-positive values.

**Comment by rofrischmann** [comment]
Thank you =)

**Comment by namjul** [comment]
Thank you for this cool project!

**Comment by rofrischmann** [comment]
=) Thanks for using it!

**Pull request merged**